primevue-gen/src/main.js

```diff
@@ -0,0 +1,22 @@
+import Vue from "vue"
+import PrimeVue from "primevue/config"
+import 'primevue/resources/themes/lara-light-teal/theme.css'
+
+import Button from 'primevue/button';
+import Dropdown from "primevue/dropdown";
+import Menubar from 'primevue/menubar';
+import Avatar from 'primevue/avatar';
+import Badge from "primevue/badge";
+
+
+window.createPrimeVueApp = function(opt) {
+  const app = Vue.createApp(opt)
+  app.use(PrimeVue)
+
+  app.component('p-dropdown', Dropdown);
+  app.component('p-button', Button);
+  app.component('p-menubar', Menubar);
+  app.component('p-avatar', Avatar);
+  app.component('p-badge', Badge);
+  return app
+}
```

Comment:
General comments about this file (`primevue-gen/src/main.js`):
- Plenty of semi-colons missing; please add them.
- The file is missing a new-line character; please add it.
- It would be nice to have all the Primevue imports and component registrations alphabetically sorted.

Reply:
> It would be nice to have all the Primevue imports and component registrations alphabetically sorted.

I think it's best to selectively import dependencies as needed during our UI update process. This approach can help reduce the size of the bundle

Reply:
There has been a misunderstanding in here. I didn't want you to import all the stuff provided by Primevue.

I meant that we could convert this...
```
import Button from 'primevue/button';
import Dropdown from "primevue/dropdown";
import Menubar from 'primevue/menubar';
import Avatar from 'primevue/avatar';
import Badge from "primevue/badge";
```
...into this:
```
import Avatar from 'primevue/avatar';
import Badge from "primevue/badge";
import Button from 'primevue/button';
import Dropdown from "primevue/dropdown";
import Menubar from 'primevue/menubar';
```
So it's alphabetically sorted, looks more organized and is easier to find stuff within that list.

Same for the component registration, converting this...
```
  app.component('p-dropdown', Dropdown);
  app.component('p-button', Button);
  app.component('p-menubar', Menubar);
  app.component('p-avatar', Avatar);
  app.component('p-badge', Badge);
```
...into this:
```
  app.component('p-avatar', Avatar);
  app.component('p-badge', Badge);
  app.component('p-button', Button);
  app.component('p-dropdown', Dropdown);
  app.component('p-menubar', Menubar);
```

Reply:
Addressed this comments in [48351b1](https://github.com/line/promgen/pull/456/commits/48351b11fe2612ccdc7f61b133241482bb75d52e)